File: compSystem/src/main/debug/java/com/effective/android/component/system/TestActivity.java
package com.effective.android.component.system;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.effective.android.base.util.FragmentUtils;
import com.plugin.component.ComponentManager;
import com.plugin.component.SdkManager;


public class TestActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.system__activity__main);
        ComponentManager.init(getApplication());
        ComponentSystemSdk systemSdk = SdkManager.getSdk(ComponentSystemSdk.class);
        FragmentUtils.replace(getSupportFragmentManager(), R.id.id_fragment_container, systemSdk.getMainFragment(), "");
    }
}

File: compSystem/src/main/debug/java/com/effective/android/component/system/TestActivity.kt
package com.effective.android.component.system

import android.os.Bundle

import com.effective.android.base.util.FragmentUtils
import com.effective.android.debug.DebugActivity
import com.plugin.component.ComponentManager
import com.plugin.component.SdkManager


class TestActivity : DebugActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        ComponentManager.init(application)
        val systemSdk = SdkManager.getSdk(ComponentSystemSdk::class.java)
        FragmentUtils.replace(supportFragmentManager, R.id.id_fragment_container, systemSdk!!.getMainFragment(), "")
    }
}
